internal/exec/terraform_outputs.go

```diff
@@ -296,19 +297,15 @@ func GetTerraformOutput(
 
 	sections, err := ExecuteDescribeComponent(component, stack, true, true, nil)
 	if err != nil {
-		if atmosConfig.Logs.Level == u.LogLevelTrace || atmosConfig.Logs.Level == u.LogLevelDebug {
-			fmt.Printf("\r✗ %s\n", message)
-		}
-		l.Fatal("Failed to describe the component", "component", component, "stack", stack, "error", err)
+		u.PrintMessage(fmt.Sprintf("\r✗ %s\n", message))
+		l.Fatal("Failed to describe the component", "component", component, "stack", stack, errorKey, err)
 	}
 
 	// Check if the component in the stack is configured with the 'static' remote state backend, in which case get the
 	// `output` from the static remote state instead of executing `terraform output`
 	remoteStateBackendStaticTypeOutputs, err := GetComponentRemoteStateBackendStaticType(sections)
 	if err != nil {
-		if atmosConfig.Logs.Level == u.LogLevelTrace || atmosConfig.Logs.Level == u.LogLevelDebug {
-			fmt.Printf("\r✗ %s\n", message)
-		}
+		u.PrintMessage(fmt.Sprintf("\r✗ %s\n", message))
```

Comment:
Does this go to stderr?

Reply:
>Does this go to stderr?

https://github.com/cloudposse/atmos/blob/main/tests/test-cases/atmos-functions.yaml#L21-L26
Based on above test case we do not add the line to stderr.

Reply:
Ok, this further drives home the point that we should have an output function named after its purpose, because engineers don't understand generally when to use stderr versus stdout.

Maybe PrintData (stdout), and PrintConsole (stderr).

Reply:
And we need a good description of each function and when to use it